chart/kubeapps/values.yaml

```diff
@@ -1581,6 +1581,14 @@ pinnipedProxy:
   ## @param pinnipedProxy.defaultPinnipedAPISuffix Specify the (default) API suffix
   ##
   defaultPinnipedAPISuffix: pinniped.dev
+  ## @param pinnipedProxy.TLSSecret Specify an optional TLS secret with which to proxy requests
```

Comment:
I guess we have to run the readmenator to add those params to the chart docs.

Reply:
Yeah - I'll do that. It does now get run automatically upstream, but no need to wait.

Reply:
Yes, fortunately, there's no need anymore, but it's nice to have our chart up to date, especially if we, want to give a try to the brand new TLS config. Thanks!